malebranche/client/logger/logger.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import (
+    annotations,
+)
+
+import logging
+from logging import (
+    Handler,
+)
+
+from malebranche.client.context import (
+    ContextManager,
+)
+
+from .filter import (
+    MalebrancheLogFilter,
+)
+from .http_avro_handler import (
+    HttpAvroHandler,
+)
+
+
+class Logger:
+    def __init__(
+        self,
+        module: str,
+        context: ContextManager,
```

Comment:
I reviewed this class and I recommend you to rename it adding a more specific name (as Python already uses the term `ContextManager` for `with something:` clauses.

Reply:
This's something I'm planning to refactor in future issues